website/app/styles/pages/application/banner.scss

```
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// BANNER
+
+@use "../../breakpoints" as breakpoint;
+
+.doc-page-banner {
+  position: relative;
+  z-index: 2;
+  display: flex;
+  grid-area: banner;
+  gap: 6px;
+  align-items: center;
+  justify-content: center;
+  padding: 16px 12px;
```

Upon sharing this with the other designers, they suggested reducing the top/bottom padding to slightly reduce the prominence since the banner is on every page. 

```suggestion
  padding: 8px 12px;
```

---

Updated